host_core/mix.exs

```diff
@@ -1,7 +1,7 @@
 defmodule HostCore.MixProject do
   use Mix.Project
 
-  @app_vsn "0.58.3"
+  @app_vsn "0.58.4"
```

Comment:
Make sure to bump this version in `wasmcloud_host` and in the Chart as well

Reply:
@autodidaddict I think this can just be `0.58.3` since we never released it, looks like it was bumped separately before